components/automate-ui/src/app/pages/project/details/project-details.component.ts

```diff
@@ -1,38 +1,40 @@
-import { Component, OnDestroy } from '@angular/core';
+import { Component, OnInit, OnDestroy } from '@angular/core';
 import { FormBuilder, FormGroup, Validators } from '@angular/forms';
 import { Store } from '@ngrx/store';
 import { Subject, combineLatest } from 'rxjs';
-import { identity } from 'lodash/fp';
-import { find as _find } from 'lodash';
 import { filter, map, pluck, takeUntil } from 'rxjs/operators';
+import { identity, some } from 'lodash/fp';
 
 import { NgrxStateAtom } from 'app/ngrx.reducers';
 import { routeParams } from 'app/route.selectors';
+import { Regex } from 'app/helpers/auth/regex';
 import { EntityStatus, loading } from 'app/entities/entities';
 import {
   getStatus, updateStatus, projectFromRoute
 } from 'app/entities/projects/project.selectors';
 import { Project } from 'app/entities/projects/project.model';
 import { GetProject, UpdateProject } from 'app/entities/projects/project.actions';
 import { GetRulesForProject, DeleteRule } from 'app/entities/rules/rule.actions';
-import { Rule } from 'app/entities/rules/rule.model';
+import { Rule, RuleStatus } from 'app/entities/rules/rule.model';
 import {
   allRules
 } from 'app/entities/rules/rule.selectors';
 
+type ProjectTabNames = 'rules' | 'details';
```

Comment:
Another string literal type for the tabs on the page--suggest we use this for all the pages with tabs.

Reply:
Oo I'll update team details